modules/ROOT/pages/assigning-permissions.adoc

```diff
@@ -0,0 +1,430 @@
+= Assigning Permissions 
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:page-aliases: assigning-administrator-permissions.adoc
+
+Anypoint Platform comes with predefined permissions that you can set in Anypoint Access Management to provide users with different access levels. You configure the permissions for administrators and Exchange viewers. 
```

Comment:
```suggestion
Anypoint Platform comes with predefined permissions that you can set in Anypoint Platform access management to provide users with different access levels. You configure the permissions for administrators and Exchange viewers. 
```

Reply:
https://confluence.internal.salesforce.com/display/MTDT/Term+Presentation+Reference#:~:text=control%20list%20(ACL)-,access%20management,-Not%20a%20title